video/out/gpu/video.c

```diff
@@ -3524,7 +3579,8 @@ static bool check_dumb_mode(struct gl_video *p)
     // otherwise, use auto-detection
     if (o->correct_downscaling || o->linear_downscaling ||
         o->linear_upscaling || o->sigmoid_upscaling || o->interpolation ||
-        o->blend_subs || o->deband || o->unsharp)
+        o->blend_subs || o->deband || o->unsharp ||
+        o->dither_algo == DITHER_ERROR_DIFFUSION)
```

Comment:
Error diffusion probably works in dumb mode as well

Reply:
It probably wont. It requires compute shader capability, and an intermediate FBO texture. Both are not available in dumb mode. The changes to `pass_dither()` actually called `finish_pass_tex()` twice for such purpose.

Reply:
They're still usable in "voluntary dumb mode". The only time you need to force-disabled this feature is if compute shaders / FBOs are unavailable, but that check already happens elsewhere.

Reply:
okay, I understand your point now. pushed another commit to enable error diffusion in dumb mode.